php/ApiHandler.php

```diff
@@ -256,43 +256,57 @@ public function verify($account_id = '')
 		}
 
 		$status = wp_remote_retrieve_response_code($verification);
+		$body = wp_remote_retrieve_body($verification);
 
-		switch ($status) {
-			case 400:
-			case 403:
-				return new WP_Error(
-					'verify-failed-' . $status,
-					__('Could not verify API and Private keys, please confirm the provided keys.', 'trustedlogin-connector')
-				);
-				break;
-			case 404:
-				return new WP_Error(
-					'verify-failed-404',
-					__('Account not found, please check the ID provided.', 'trustedlogin-connector')
-				);
-				break;
-			case 405:
-				return new WP_Error(
-					'verify-failed-405',
-					sprintf(
+		$body = json_decode($body);
+
+		if ( $status > 399 ) {
+			switch ( $status ) {
+				case 402:
+					return new WP_Error(
+						'verify-failed-402',
+						__( 'You do not have a valid TrustedLogin subscription.', 'trustedlogin-connector' )
+					);
+					break;
```

Comment:
@mgratch Let's move down so it's in numerical order.

Reply:
But the next case covers 400 & 403 status codes.

Reply:
I am going to leave this, but instead of merging with develop, I am merging into a mini-release branch so we can do 1 last round of checks when all the other PRs are pushed up.